docs/tutorial.md

```diff
@@ -436,7 +435,7 @@ In Mineflayer, callbacks are often used to handle errors.
 
 ```js
 bot.consume((error) => {
-  if (error) {
+  if (error != undefined) { // If no error occored, 'error' will be undefined. If it is, an error occured.
```

Comment:
most likely it'll be `null`. Previous check was better as it covers both undefined and null and does not involve an equality operator (You should avoid those and use identity operators.)